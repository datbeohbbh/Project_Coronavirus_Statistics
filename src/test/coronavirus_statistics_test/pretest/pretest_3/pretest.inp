6 11
q 1 3 22
? 4 1
q 2 3 7
? 4 1
q 2 3 1
q 1 4 1
? 4 2
? 4 1
? 7
? 7
? 4 2