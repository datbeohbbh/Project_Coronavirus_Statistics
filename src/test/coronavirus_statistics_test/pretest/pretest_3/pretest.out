Query ? 4 1 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 22
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 15
-> Has recovered : 7
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 14
-> Has recovered : 8
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 1
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 2 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 14
-> Has recovered : 8
-> Was dead : 0

